VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/individual-participant-status-list/individual-participant-status-list.component.html

```diff
@@ -1,154 +1,175 @@
+    <div *ngIf='wingers.length > 0'>
+        <div class='govuk-grid-row'>
+            <div class='govuk-grid-column-full'>
+                <h2 class='govuk-heading-s'>{{'individual-participant-status-list.wingers' | translate}}
+                    ({{ wingers.length }})
+                </h2>
+            </div>
+        </div>
+        <div class='vh-border'>
+            <div *ngFor='let participant of wingers; let lastItem = last'
+                 class='govuk-grid-row govuk-!-margin-bottom-1'>
+                <div id='p-row-{{ participant.id }}-panel-member'
+                     class='govuk-grid-column-full'>
+                    <p id='p-{{ participant.id }}-name-panel-member'
+                       class='govuk-heading-s govuk-!-margin-bottom-0'>{{ participant.name }}</p>
+                    <div class='vh-line'
+                         *ngIf='!lastItem'></div>
+                </div>
+            </div>
         </div>
-      </div>
     </div>
-  </div>
 
-  <div *ngIf="panelMembers.length > 0">
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full">
-        <h2 class="govuk-heading-s">{{'individual-participant-status-list.panel-members' | translate}} ({{
-          panelMembers.length }})</h2>
-      </div>
+    <div *ngIf='panelMembers.length > 0'>
+        <div class='govuk-grid-row'>
+            <div class='govuk-grid-column-full'>
+                <h2 class='govuk-heading-s'>{{'individual-participant-status-list.panel-members' | translate}} ({{
+                    panelMembers.length }})</h2>
+            </div>
+        </div>
+        <div class='vh-border'>
+            <div *ngFor='let participant of panelMembers; let lastItem = last'
+                 class='govuk-grid-row govuk-!-margin-bottom-1'>
+                <div id='p-row-{{ participant.id }}-panel-member'
+                     class='govuk-grid-column-full'>
+                    <p id='p-{{ participant.id }}-name-panel-member'
+                       class='govuk-heading-s govuk-!-margin-bottom-0'>{{ participant.name }}</p>
+                    <div class='vh-line'
+                         *ngIf='!lastItem'></div>
+                </div>
+            </div>
+        </div>
     </div>
-    <div class="vh-border">
-      <div *ngFor="let participant of panelMembers; let lastItem = last"
-        class="govuk-grid-row govuk-!-margin-bottom-1">
-        <div id="p-row-{{ participant.id }}-panel-member"
-          class="govuk-grid-column-full">
-          <p id="p-{{ participant.id }}-name-panel-member"
-            class="govuk-heading-s govuk-!-margin-bottom-0">{{ participant.name }}</p>
-          <div class="vh-line"
-            *ngIf="!lastItem"></div>
+    <div *ngIf='staffMembers.length > 0'>
+        <div class='govuk-grid-row'>
+            <div class='govuk-grid-column-full'>
+                <h2 class='govuk-heading-s'>{{'individual-participant-status-list.staff-members' | translate}} </h2>
```

Comment:
not sure if  need show numbers of the staff members for now like other type particilant

Reply:
screenshot added ny neil have no number